examples/learn/local_models/ollama.py

```diff
@@ -0,0 +1,41 @@
+import asyncio
+
+from mirascope.core import openai
+from pydantic import BaseModel
+from openai import AsyncOpenAI
+
+# Follow the link to see what features of openai are supported
+# https://github.com/ollama/ollama/blob/main/docs/openai.md
+ollama_client = AsyncOpenAI(
+    base_url = 'http://localhost:11434/v1', # your ollama endpoint
+    api_key='unused', # required by openai, but unused
+)
+
+
+class Book(BaseModel):
+    title: str
+    author: str
+
+
+@openai.call("llama3.2", response_model=Book, client=ollama_client)
+async def extract_book(text: str) -> str:
```

Comment:
nit: these examples should be synchronous (only the Async section should have async)